py/test/selenium/webdriver/common/element_attribute_tests.py

```diff
@@ -64,6 +64,11 @@ def test_should_return_the_value_of_the_disabled_attribute_as_false_if_not_set(d
     pElement = driver.find_element(By.ID, "peas")
     assert pElement.get_attribute("disabled") is None
     assert pElement.is_enabled()
+    
+def test_should_return_false_for_enabled_element_with_pointer_events_set_to_none(driver, pages):
+    pages.load("formPage.html")
+    inputElement = driver.find_element(By.ID, "pointerEventsNone")
+    assert not inputElement.is_enabled()
```

Comment:
This is wrong:
* The element is enabled.
* The element is only not clickable (does not receive pointer events)
* The element can still be triggered by keyboard

Please read https://github.com/SeleniumHQ/selenium/issues/14427#issuecomment-2307123929

Reply:
https://github.com/SeleniumHQ/selenium/issues/14427#issuecomment-2307256232